feature/sref: increase walltime for precip stats job

BinbinZhou-NOAA opened the pull request:
## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

BinbinZhou-NOAA committed:
increase walltime for precip stats job

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/sref

BinbinZhou-NOAA committed:
remove logo hard-coded path in settings.py

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/sref

ShelleyMelchior-NOAA commented:
Hi @BinbinZhou-NOAA , I ran the 4 sref plots jobs. Please review
logfiles:
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_precip_spatial_plots.o78117572`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_cnv_past90days_plots.o78117569`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_precip_past90days_plots.o78117571`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_grid2obs_past90days_plots.o78117570`
DATA=`/lfs/h2/emc/stmp/shelley.melchior/evs/tmpnwprd/jevs_mesoscale_sref_precip_spatial_plots.78117572.cbqs01`
DATA=`/lfs/h2/emc/stmp/shelley.melchior/evs/tmpnwprd/jevs_mesoscale_sref_cnv_past90days_plots.78117569.cbqs01`
DATA=`/lfs/h2/emc/stmp/shelley.melchior/evs/tmpnwprd/jevs_mesoscale_sref_precip_past90days_plots.78117571.cbqs01`
DATA=`/lfs/h2/emc/stmp/shelley.melchior/evs/tmpnwprd/jevs_mesoscale_sref_grid2obs_past90days_plots.78117570.cbqs01`
COMOUT=`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/mesoscale/atmos.20230822`

BinbinZhou-NOAA commented:
Shelley,

  All of the 4 tar files are fine, buth there is a timing issue for cnv tar
file:

In your cnv driver, the VDATE=20230822, but it was run on 20230825, so in
the job file
https://github.com/NOAA-EMC/EVS/blob/develop/jobs/mesoscale/plots/JEVS_MESOSCALE_PLOTS
,
line 55 set the VDATE=20230823 (since PDY=20230825).
This cause the output tar file name
became evs.plots.sref.cnv.past90days.v20230823.tar and stored it
in COMOUT=..../atmos.20230823 directory instead of other 3 tar files in
atmos.20230822.
The cnv tar file itself is good.

Binbin

On Thu, Aug 24, 2023 at 10:24 PM Shelley Melchior ***@***.***>
wrote:

> Hi @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> , I ran the 4
> sref plots jobs. Please review
> logfiles:
>
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_precip_spatial_plots.o78117572
>
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_cnv_past90days_plots.o78117569
>
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_precip_past90days_plots.o78117571
>
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_grid2obs_past90days_plots.o78117570
> DATA=
> /lfs/h2/emc/stmp/shelley.melchior/evs/tmpnwprd/jevs_mesoscale_sref_precip_spatial_plots.78117572.cbqs01
> DATA=
> /lfs/h2/emc/stmp/shelley.melchior/evs/tmpnwprd/jevs_mesoscale_sref_cnv_past90days_plots.78117569.cbqs01
> DATA=
> /lfs/h2/emc/stmp/shelley.melchior/evs/tmpnwprd/jevs_mesoscale_sref_precip_past90days_plots.78117571.cbqs01
> DATA=
> /lfs/h2/emc/stmp/shelley.melchior/evs/tmpnwprd/jevs_mesoscale_sref_grid2obs_past90days_plots.78117570.cbqs01
> COMOUT=
> /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/mesoscale/atmos.20230822
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/207#issuecomment-1692665031>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFOISJS3EKG7CPK6VE3XXAEHFANCNFSM6AAAAAA3VW7MGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Thanks @BinbinZhou-NOAA . Glad the plots are all good, even if saved in the wrong location. I find it interesting that the problem happened, though, since `JEVS_MESOSCALE_PLOTS` accepts an upstream override for VDATE.

`if [ $MODELNAME = sref ] ; then`
`    export VDATE=${VDATE:-$PDYm2}`

ShelleyMelchior-NOAA commented:
@BinbinZhou-NOAA , for completeness, I re-ran sref cnv past90days again, this time not setting VDATE. The processing assigned 20230823 (PDYm2) and the output tar file wrote to the correct directory.
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_cnv_past90days_plots.o78216388`
COMOUT=`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/mesoscale/atmos.20230823`

If this looks OK then I think this PR is complete.

BinbinZhou-NOAA commented:
Shelley,

  Yes, the cnv plot tar file is good.

Binbin

On Fri, Aug 25, 2023 at 4:58 PM Shelley Melchior ***@***.***>
wrote:

> @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> , for completeness,
> I re-ran sref cnv past90days again, this time not setting VDATE. The
> processing assigned 20230823 (PDYm2) and the output tar file wrote to the
> correct directory.
> logfile:
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_cnv_past90days_plots.o78216388
> COMOUT=
> /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/mesoscale/atmos.20230823
>
> If this looks OK then I think this PR is complete.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/207#issuecomment-1693921817>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFOM3VXKTWN625KN7YDXXEGVDANCNFSM6AAAAAA3VW7MGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Great, @BinbinZhou-NOAA . Please complete a sync fork for this branch and then I can merge the PR.

LoganDawson-NOAA commented:
@ShelleyMelchior-NOAA I was initially going to ask if this PR could remain open until Binbin addresses the plotting updates from me and Geoff, but I see that the initial purpose of this PR was to request more resources for a stats job. 

That resource increase should get into the parallel asap, so I'm ok with merging this PR at this stage. Will just note that we'll need one more SREF PR that includes the other plotting updates that were requested via email

BinbinZhou-NOAA commented:
Shelley,

  I'll work on some new request for the SREF plotting next week. So please
hold on the merging untill my work is done.

Logan,

  I just found the 2 logos are not in the plots after their paths were
modified as
         self.logo_left_path = f"{os.environ['FIXevs']}/logos/noaa.png"
        self.logo_right_path =  f"{os.environ['FIXevs']}/logos/nws.png"

Not sure what is the problem.

Binbin



On Sun, Aug 27, 2023 at 10:05 AM Logan Dawson ***@***.***>
wrote:

> @ShelleyMelchior-NOAA <https://github.com/ShelleyMelchior-NOAA> I was
> initially going to ask if this PR could remain open until Binbin addresses
> the plotting updates from me and Geoff, but I see that the initial purpose
> of this PR was to request more resources for a stats job.
>
> That resource increase should get into the parallel asap, so I'm ok with
> merging this PR at this stage. Will just note that we'll need one more SREF
> PR that includes the other plotting updates that were requested via email
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/207#issuecomment-1694677612>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFOJXECALGP3WVHYFBLXXNH3RANCNFSM6AAAAAA3VW7MGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA commented:
Logan,

  I found the problem. In my plot drivers, FIXevs was not defined.
After added it, the 2 logos appear.

Shelley,

   The parameter FIXevs in the narre plot drivers is missing. This causes
the 2 logos to be missing
in the narre plots.  Could you please add following one line
     export
FIXevs=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix
into  following 2 driver scripts (in
.../EVS/dev/drivers/scripts/narre/plots)
    evs_narre_past31days_plots.sh
    jevs_narre_past90days_plots.sh

 So I have not to submit a new PR for narre.

Thanks!

Binbin



On Sun, Aug 27, 2023 at 2:13 PM Binbin Zhou - NOAA Affiliate <
***@***.***> wrote:

> Shelley,
>
>   I'll work on some new request for the SREF plotting next week. So please
> hold on the merging untill my work is done.
>
> Logan,
>
>   I just found the 2 logos are not in the plots after their paths were
> modified as
>          self.logo_left_path = f"{os.environ['FIXevs']}/logos/noaa.png"
>         self.logo_right_path =  f"{os.environ['FIXevs']}/logos/nws.png"
>
> Not sure what is the problem.
>
> Binbin
>
>
>
> On Sun, Aug 27, 2023 at 10:05 AM Logan Dawson ***@***.***>
> wrote:
>
>> @ShelleyMelchior-NOAA <https://github.com/ShelleyMelchior-NOAA> I was
>> initially going to ask if this PR could remain open until Binbin addresses
>> the plotting updates from me and Geoff, but I see that the initial purpose
>> of this PR was to request more resources for a stats job.
>>
>> That resource increase should get into the parallel asap, so I'm ok with
>> merging this PR at this stage. Will just note that we'll need one more SREF
>> PR that includes the other plotting updates that were requested via email
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/NOAA-EMC/EVS/pull/207#issuecomment-1694677612>, or
>> unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQBMPFOJXECALGP3WVHYFBLXXNH3RANCNFSM6AAAAAA3VW7MGM>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
>
> --
>
> Binbin Zhou
>
> Physical Scientist
>
> Lynker at NOAA/NWS/NCEP/EMC
>
> 5830 University Research Ct.
>
> College Park, MD 20740
>
> ***@***.***
>
> 301-683-3683
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/sref

BinbinZhou-NOAA committed:
export FIXevs in the plot drivers

ShelleyMelchior-NOAA committed:
Updated narre plots drivers to include export of FIXevs.

ShelleyMelchior-NOAA commented:
@BinbinZhou-NOAA , the narre plot drivers are updated to export FIXevs. I might as well include the narre_plots modulefile update in this PR, too.

ShelleyMelchior-NOAA commented:
I ran the narre plots w/ the udpated drivers and a trial w/ the plots modulefile updated to not load met and metplus. The logos appear in the graphics, but they are not positioned well.
COMOUT=`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20230826`
I have them untar'd on rzdm: https://ftp.emc.ncep.noaa.gov/users/smelchior/EVS/narre/

ShelleyMelchior-NOAA committed:
Updating the plots modulefile to not load met and metplus.

ShelleyMelchior-NOAA committed:
Updating the plots modulefile to not load met and metplus.

BinbinZhou-NOAA commented:
Shelley,

  Yes, they are too close to the center but not easy to adjust.

Binbin


On Sun, Aug 27, 2023 at 3:38 PM Shelley Melchior ***@***.***>
wrote:

> I ran the narre plots w/ the udpated drivers and a trial w/ the plots
> modulefile updated to not load met and metplus. The logos appear in the
> graphics, but they are not positioned well.
> COMOUT=
> /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/narre/atmos.20230826
> I have them untar'd on rzdm:
> https://ftp.emc.ncep.noaa.gov/users/smelchior/EVS/narre/
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/207#issuecomment-1694744530>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFP34FXDQ3T7HS2SBZ3XXOO3ZANCNFSM6AAAAAA3VW7MGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/sref

BinbinZhou-NOAA committed:
Separate validation times/thresholds in the cape and cnv plots

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/sref

BinbinZhou-NOAA committed:
separate valid times in precip and grid2obs plots

ShelleyMelchior-NOAA commented:
Hi @BinbinZhou-NOAA , let me know when this PR is ready for more testing.

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/sref

BinbinZhou-NOAA commented:
Shelley,

  The SREF updates have been done and its PR is ready for more testing.

Binbin



On Fri, Sep 1, 2023 at 7:30 PM Shelley Melchior ***@***.***>
wrote:

> Hi @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> , let me know
> when this PR is ready for more testing.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/207#issuecomment-1703501135>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFK5I5JFOTY7USTL7ZDXYJVYTANCNFSM6AAAAAA3VW7MGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
@BinbinZhou-NOAA , I re-ran the plots jobs and updated DATA to be DATAROOT in my local clone of your fork. Please review.
```
$ ll /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/mesoscale/atmos.20230903/
total 86456
-rw-r--r-- 1 shelley.melchior emc 20858880 Sep  5 02:22 evs.plots.sref.cape.past90days.v20230903.tar
-rw-r--r-- 1 shelley.melchior emc 44820480 Sep  5 02:18 evs.plots.sref.cnv.past90days.v20230903.tar
-rw-r--r-- 1 shelley.melchior emc 65382400 Sep  5 02:27 evs.plots.sref.grid2obs.past90days.v20230903.tar
-rw-r--r-- 1 shelley.melchior emc 22845440 Sep  5 02:18 evs.plots.sref.precip.past90days.v20230903.tar
-rw-r--r-- 1 shelley.melchior emc    10240 Sep  5 02:14 evs.plots.sref.precip.spatial.map.v20230903.tar
```
The precip spatial tar is empty. 
logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_*_plots.o9592713*`

BinbinZhou-NOAA commented:
Shelley,

  All of the sref plot tar files are godd except the spatial tar file.
I checked its *.o file and the log file  and found the reason is hte 24h
APCP accumulated files do not exit in the parallel stat
directory:

*WARNING:
/lfs/h2/emc/stmp/shelley.melchior/evs/tmpnwprd/jevs_mesoscale_sref_precip_spatial_plots.95927137.dbqs01/
grid2grid_plots/data/sref/sref_precip_24hrAccum_init2023090212_fhr024.nc
does not exist, not making plot*

So if you use my stat files as input, it should be working.
Thanks!

Binbin


On Mon, Sep 4, 2023 at 10:29 PM Shelley Melchior ***@***.***>
wrote:

> @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> , I re-ran the
> plots jobs and updated DATA to be DATAROOT in my local clone of your fork.
> Please review.
>
> $ ll /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/mesoscale/atmos.20230903/
> total 86456
> -rw-r--r-- 1 shelley.melchior emc 20858880 Sep  5 02:22 evs.plots.sref.cape.past90days.v20230903.tar
> -rw-r--r-- 1 shelley.melchior emc 44820480 Sep  5 02:18 evs.plots.sref.cnv.past90days.v20230903.tar
> -rw-r--r-- 1 shelley.melchior emc 65382400 Sep  5 02:27 evs.plots.sref.grid2obs.past90days.v20230903.tar
> -rw-r--r-- 1 shelley.melchior emc 22845440 Sep  5 02:18 evs.plots.sref.precip.past90days.v20230903.tar
> -rw-r--r-- 1 shelley.melchior emc    10240 Sep  5 02:14 evs.plots.sref.precip.spatial.map.v20230903.tar
>
> The precip spatial tar is empty.
> logfiles:
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_*_plots.o9592713*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/207#issuecomment-1705860616>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFO7WCRRB6MSSYRFIETXY2FAHANCNFSM6AAAAAA3VW7MGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA committed:
Change DATA to DATAROOT

ShelleyMelchior-NOAA commented:
Thanks, @BinbinZhou-NOAA  -- I will re-run precip spatial pointing to your stats. Then I think we are in good shape.

BinbinZhou-NOAA commented:
Shelley,

  I have changed DATA to DATAROOT in both stats and plots driver scripts
for all regional ensembles
and updated them in github.

Binbin

On Tue, Sep 5, 2023 at 10:34 AM Shelley Melchior ***@***.***>
wrote:

> Thanks, @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> -- I will
> re-run precip spatial pointing to your stats. Then I think we are in good
> shape.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/207#issuecomment-1706743537>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFKSOU4WRP2Z33HO3HLXY42AFANCNFSM6AAAAAA3VW7MGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

ShelleyMelchior-NOAA commented:
Thanks @BinbinZhou-NOAA . I also re-ran precip spatial pointing to your stats. It ran to completion and created a non-empty tar file.
```
$ ll /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/mesoscale/atmos.20230903/                                                              
total 150508
-rw-r--r-- 1 shelley.melchior emc 20858880 Sep  5 02:22 evs.plots.sref.cape.past90days.v20230903.tar
-rw-r--r-- 1 shelley.melchior emc 44820480 Sep  5 02:18 evs.plots.sref.cnv.past90days.v20230903.tar
-rw-r--r-- 1 shelley.melchior emc 65382400 Sep  5 02:27 evs.plots.sref.grid2obs.past90days.v20230903.tar
-rw-r--r-- 1 shelley.melchior emc 22845440 Sep  5 02:18 evs.plots.sref.precip.past90days.v20230903.tar
-rw-r--r-- 1 shelley.melchior emc   204800 Sep  5 15:06 evs.plots.sref.precip.spatial.map.v20230903.tar
```
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_precip_spatial_plots.o95960333`
I think we are in good shape. I will start the code review.
Also, please be sure to do another sync fork.

BinbinZhou-NOAA commented:
Shelley,

  Now the spatial tar file is good.
Sure I'll symc my feature/sref fork

Thanks!


Binbin

On Tue, Sep 5, 2023 at 11:12 AM Shelley Melchior ***@***.***>
wrote:

> Thanks @BinbinZhou-NOAA <https://github.com/BinbinZhou-NOAA> . I also
> re-ran precip spatial pointing to your stats. It ran to completion and
> created a non-empty tar file.
>
> $ ll /lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/mesoscale/atmos.20230903/
> total 150508
> -rw-r--r-- 1 shelley.melchior emc 20858880 Sep  5 02:22 evs.plots.sref.cape.past90days.v20230903.tar
> -rw-r--r-- 1 shelley.melchior emc 44820480 Sep  5 02:18 evs.plots.sref.cnv.past90days.v20230903.tar
> -rw-r--r-- 1 shelley.melchior emc 65382400 Sep  5 02:27 evs.plots.sref.grid2obs.past90days.v20230903.tar
> -rw-r--r-- 1 shelley.melchior emc 22845440 Sep  5 02:18 evs.plots.sref.precip.past90days.v20230903.tar
> -rw-r--r-- 1 shelley.melchior emc   204800 Sep  5 15:06 evs.plots.sref.precip.spatial.map.v20230903.tar
>
> logfile:
> /lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_mesoscale_sref_precip_spatial_plots.o95960333
> I think we are in good shape. I will start the code review.
> Also, please be sure to do another sync fork.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/207#issuecomment-1706811832>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFORIRCTYDTNPNZL6IDXY46PBANCNFSM6AAAAAA3VW7MGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683

BinbinZhou-NOAA committed:
Merge branch 'NOAA-EMC:develop' into feature/sref

ShelleyMelchior-NOAA reviewed (approved):
This PR is approved for merge.

BinbinZhou-NOAA commented:
Shelley,

  After this merge, Logan suggested another modification to remove cartopy
hard-coded path in the spatial map py file.
I did this for both sref and href.  For href, this removal can be put in
the PR (still open)
But for SREF, the feature/sref has just been merged, so I won't  open a new
PR for this change.
Please do it for me:  that is, change line 106 in the file
https://github.com/NOAA-EMC/EVS/blob/develop/ush/mesoscale/ush_sref_plot_precip_py/sref_atmos_plots_precip_spatial_map.py
from  config['data_dir'] = '/apps/ops/prod/data/cartopy'
to      config['data_dir'] = f"{os.environ['cartopyDataDir']}"

Thanks!

Binbin


On Tue, Sep 5, 2023 at 11:20 AM Shelley Melchior ***@***.***>
wrote:

> Merged #207 <https://github.com/NOAA-EMC/EVS/pull/207> into develop.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NOAA-EMC/EVS/pull/207#event-10285117259>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBMPFLYRJN72BQQV3SN4U3XY47MLANCNFSM6AAAAAA3VW7MGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Binbin Zhou

Physical Scientist

Lynker at NOAA/NWS/NCEP/EMC

5830 University Research Ct.

College Park, MD 20740

***@***.***

301-683-3683
